ext/hermann/hermann_lib.c

```diff
@@ -359,22 +353,48 @@ static void consumer_consume_stop_callback(void *ptr) {
 }
 #endif
 
+void *consumer_recv_msg(void *ptr)
+{
+	rd_kafka_message_t *ret;
+	HermannInstanceConfig *consumerConfig = (HermannInstanceConfig *) ptr;
+
+	ret = rd_kafka_consume(consumerConfig->rkt, consumerConfig->partition, 100);
```

Comment:
can this timeout constant be defined elsewhere as a constant? I don't think it makes sense to expose up to the Ruby layer, but I have a fear of magic numbers in fnuctions :P